src/lib/select/select.spec.ts

```diff
@@ -1930,52 +1881,52 @@ describe('MatSelect', () => {
       let fixture: ComponentFixture<BasicSelect>;
       let select: HTMLElement;
 
-      beforeEach(() => {
+      beforeEach(fakeAsync(() => {
         fixture = TestBed.createComponent(BasicSelect);
         fixture.detectChanges();
         select = fixture.debugElement.query(By.css('mat-select')).nativeElement;
-      });
+      }));
 
-      it('should set the role of the select to listbox', () => {
+      it('should set the role of the select to listbox', fakeAsync(() => {
```

Comment:
Most of these tests aren't doing anything async, why do we have to bring them into the `fakeAsync` zone (aside from consistency)?

Reply:
I just added it to everything for safety. Some of the tests that weren't marked as async actually were and could leak async operations into later tests. I figured this way we guarantee that won't happen